### DISCUSSION TITLE
Any way to detect Tailwind css?

### DISCUSSION THREAD

**Discussion by uurcank** [tailwindlabs/tailwindcss#9312] [Help]
What would be the easy way to detect if a page is using tailwind? Does library inject any meta fields to the page or something similar that can be useful for this purpose?

**Comment by wongjn** [comment]
Tailwind does not affect the page in any discernible way – it touches the CSS only. So you might be able to detect Tailwind by looking at the page's stylesheets, perhaps see if there are any `--tw-` CSS variables? Another option would be to look for the license header `/* ! tailwindcss v3.1.8 | MIT License | https://tailwindcss.com */` or similar, although I believe some bundlers extract license headers to separate files.
